tests/menus/test_frequently_used_context_menu.py

```diff
@@ -0,0 +1,120 @@
+import platform
+from time import sleep
+
+import pytest
+from selenium.webdriver import Firefox
+
+from modules.browser_object import ContextMenu, Navigation
+from modules.page_object import ExamplePage
+from modules.util import BrowserActions, Utilities
+
+
+@pytest.mark.unstable
+def test_save_page_as(driver: Firefox):
+    """
+    C2637623.1: save page as
+    """
+    try:
+        from pynput.keyboard import Key
+    except ModuleNotFoundError:
+        pytest.skip("Could not load pynput")
+    # create objects
+    context_menu = ContextMenu(driver)
+    driver.get("https://example.com")
+    example_page = ExamplePage(driver)
+    nav = Navigation(driver)
+    util = Utilities()
+    ba = BrowserActions(driver)
+
+    # right click something that is not a hyperlink
+    title_header = example_page.get_element("title-header")
+    context_menu.context_click_element(title_header)
+
+    with driver.context(driver.CONTEXT_CHROME):
+        save_page_as = context_menu.get_context_item("context-menu-save-page-as")
```

Comment:
It seems weird that we have to explicitly call change the context here, is there a way to push that functionality into the BOM method `get_context_item`?

Reply:
I added it in!